ext/vulnsrc/manual/manual.go

```diff
@@ -994,10 +994,10 @@ var Vulnerabilities = []database.Vulnerability{
 					"Vectors":             "",
 				},
 				"CVSSv3": map[string]interface{}{
-					"ExploitabilityScore": 1.6,
-					"ImpactScore":         6.0,
-					"Score":               8.3,
-					"Vectors":             "CVSS:3.1/AV:N/AC:H/PR:N/UI:R/S:C/C:H/I:H/A:H",
+					"ExploitabilityScore": 3.9,
+					"ImpactScore":         3.6,
+					"Score":               7.5,
+					"Vectors":             "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
```

Comment:
I have no idea where they got this score from. I can't find it in their repo https://git.launchpad.net/ubuntu-cve-tracker/tree/active/ nor NVD https://nvd.nist.gov/vuln/detail/CVE-2024-47176

I wonder if this score changed? I can handle the other distros in a followup PR so this isn't blocked